### PR TITLE
Advertise first Knative Virtual Office Hour 2022/12/09

### PR DESCRIPTION
**Pull request by aslom**
## Proposed Changes 

- Advertise first Knative Virtual Office Hour / Contrib Fest 2022/12/09

### CODE REVIEW

**Comment by aslom** [comment]
Posted about it in Knative general slack channel

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:6]: I might call this "Virtual Office Hour" and drop the "Contrib Fest" part. I think having "open for new contributors" in the text body is okay, but I think having end-users show up just to ask questions or chat should be explicitly our goal, and adding "ContribFest" adds a bunch of pressure to become a contributor.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:13]:
```suggestion
During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed that there are people interested in using Knative or contributing to it. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is very asynchronous and joining yet another Slack and then reading channels, figuring where to ask your questions, and driving your own engagement can be a barrier to participating in the community.
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:25]:
```suggestion
- You have great idea for Knative but you do not want where to start
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:23]: This feels like a duplicate of the last one.

Maybe:

> You are trying to use Knative and trying to figure out how to do a specific thing with it.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:24]: I'd remove the "asking in Slack or GitHub", that feels like it might shame people into not attending. We should steer them _after_ they show up, rather than making them feel like they can't show up. 😁

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:47]: I think the feedback from the Maintainer Meetup was that we didn't have a community <-> contributors dialogue; I'd like to focus this more on that dialogue than on immediately moving folks into contributing.

**Review by evankanderson — COMMENTED** [review]
Thanks for organizing this, @aslom !

I'd like to focus this particularly on folks who might currently be described as "lurkers" to bring them into the conversation circle. If we discover there's a big cadre of folks who are waiting for someone to take them under their wing and steer them into being contributors, I think that would be a good _separate_ meeting.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (psschwei) [inline blog/docs/events/virtual-office-hour-12-2022.md]:
```suggestion
**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), and Paul Schweigert**

**Date: 2022-12-01**

# First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09
```

this is the new format we're using for blog headers that we switched to a little bit ago

**Review by psschwei — CHANGES_REQUESTED** [review]
You'll also need to update the blog navigation so that the post will show on the site... I'd add it to top of the "Events" section here: https://github.com/knative/docs/blob/b06502df49327bcca031a1a985e1fd5078ff8378/blog/config/nav.yml#L74

Might also want to replace the Kubecon post with it on the blog landing page: https://github.com/knative/docs/blob/b06502df49327bcca031a1a985e1fd5078ff8378/blog/docs/index.md?plain=1#L25-L29

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md:45]: Yes. When we have the blog post published I think we should send email to knative-users to invite them to the office hour for dialogue and for us to listen.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md]: Updated to the new blog format.

Let make separate PR to docs/blog/config/nav.yml and docs/blog/docs/index.md when this PR is merged?

Comment (psschwei) [reply blog/docs/events/virtual-office-hour-12-2022.md]: I'd do at least the nav.yaml one in this PR, otherwise the blog post won't be visible.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (lance) [inline blog/docs/events/virtual-office-hour-12-2022.md:9]: Just a little bit of wordsmithing:
```suggestion
During the recent KubeCon NA 2022 in Detroit we had Knative Day, Maintainer Meetup, and ContribFest that showed there are people interested in using or contributing to Knative. However, today we do not have any ongoing online meetings for people to just show up. They can join Slack but that is asynchronous and requires joining yet another Slack, then reading channels, and figuring where to ask your questions. Driving your own engagement can be a barrier to participating in the community.
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (lance) [inline blog/docs/events/virtual-office-hour-12-2022.md:11]: Agree with @evankanderson that the focus should be on the users, and less so contributors.
```suggestion
Therefore, we are organizing the first Knative Virtual Office Hour in 2022.
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (lance) [inline blog/docs/events/virtual-office-hour-12-2022.md:19]:
```suggestion
- You have great idea for Knative but you do not know where or how to share it
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:14]: We should go back and update this when we have the calendar, but there's no reason not to post the blog post now.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:37]:
```suggestion
```

I'm not sure we want to link the Contrib fest part here past the slides.

Instead, I added a couple more items for "who should join".

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:7]: Can we add a splash image, like https://unsplash.com/photos/Q0AM87PsYkE ?

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:19]:
```suggestion
- You have questions about how to build something using Knative
- You have great idea for Knative but you do not know where or how to share it
- You want to meet other people using Knative
```

**Review by evankanderson — COMMENTED** [review]
A few more small comments, but this is looking quite good, thank you!

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md:7]: Maybe one of photoa from KubeCon with Knative in background form KnativeCon or other meeting?

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md:14]: :+1:

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md:39]: OK with me - we can see if there is need for virtual contrib fest in future.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [reply blog/docs/events/virtual-office-hour-12-2022.md:7]: 👍 

That's a great idea!

**Review by evankanderson — COMMENTED** [review]
Can you:

1. Add the photo
2. Update https://github.com/knative/docs/blob/main/blog/config/nav.yml#L74 to put this at the head of that list?

**Comment by aslom** [comment]
I added blog to blog/config/nav.yml

@evankanderson  @lance @psschwei I think to use this photo: https://photos.app.goo.gl/u7Br8eV4B8wmMqje6 - I think to reduce it in size and add to github as .jpg cropped and scaled to 1200x600
virtual-office-hour-12-2022

**Comment by aslom** [comment]
Image with text:
virtual-office-hour-12-2022-with-text

**Comment by evankanderson** [comment]
> I like the picture, with one small nit: is there a way we could scrub out the purple restrooms sign?

IMO, don't let perfect be the enemy of good!

**Comment by psschwei** [comment]
> > I like the picture, with one small nit: is there a way we could scrub out the purple restrooms sign?
> 
> IMO, don't let perfect be the enemy of good!

That got posted before I saw the image with text :smile: I'm good with it too

**Comment by aslom** [comment]
Added the image to the blog - I think the blog is ready? @evankanderson @psschwei @lance

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (psschwei) [inline blog/docs/events/virtual-office-hour-12-2022.md:2]:
```suggestion
# First 🌱 Knative Virtual Office Hour

![Knative Booth](images/virtual-office-hour-12-2022-with-text.jpg)
```

I think due to how mkdocs renders the page, the first line needs to be the title, otherwise it uses the filename on the left-side menu.

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (psschwei) [inline blog/docs/events/virtual-office-hour-12-2022.md:17]:
```suggestion
**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)

**How to join:** Zoom link will be posted in Slack and knative-user mailing list
```

**Review by psschwei — COMMENTED** [review]
couple of markdown rendering nits, otherwise LGTM

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md]: Moved to safe location - hopefully mkdocs will be happy

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md]: done

**Comment by aslom** [comment]
The blog post looks good in website deploy preview: https://deploy-preview-5341--knative.netlify.app/blog/events/virtual-office-hour-12-2022/  @psschwei @evankanderson @lance

**Review by psschwei — COMMENTED** [review]
/lgtm
/approve
/hold 
(in case @lance or @evankanderson have anything else, otherwise we can unhold/merge by EOD)

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (evankanderson) [inline blog/docs/events/virtual-office-hour-12-2022.md:15]:
```suggestion
**When:** [December 9 (Friday) 11:30 AM US Eastern](https://dateful.com/convert/utc?t=430pm&d=2022-12-09)
<a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=NXFjbzVjM3NqZTU1aHNoYmtkbjA5am1uOW4ga25hdGl2ZS50ZWFtXzlxODNiZzA3cXM1YjlycnNscDVqb3I0bDZzQGc&amp;tmsrc=knative.team_9q83bg07qs5b9rrslp5jor4l6s%40group.calendar.google.com"><img border="0" alt="Add to Google Calendar" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>
```

Comment (evankanderson) [reply blog/docs/events/virtual-office-hour-12-2022.md:15]: (I created a calendar entry)

Comment (aslom) [reply blog/docs/events/virtual-office-hour-12-2022.md:15]: :+1:

**Comment by aslom** [comment]
Please review and lgtm after calendar link was added to have the blog post published in the website. I tested calendar link from https://deploy-preview-5341--knative.netlify.app/blog/events/virtual-office-hour-12-2022/

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (psschwei) [inline blog/docs/events/virtual-office-hour-12-2022.md:18]:
```suggestion
**How to join:** Zoom: https://zoom.us/j/94310596244
```

File: blog/docs/events/virtual-office-hour-12-2022.md
Comment (psschwei) [inline blog/docs/events/virtual-office-hour-12-2022.md:18]: My bad, I gave you a non-clickable link :cry: 

```suggestion
**How to join:** Zoom: [https://zoom.us/j/94310596244](https://zoom.us/j/94310596244)
```

**Comment by aslom** [comment]
@psschwei @evankanderson @lance final version?

**Review by evankanderson — COMMENTED** [review]
/lgtm

**Review by evankanderson — APPROVED** [review]

**Pull request merged**
